src/Datadog.Trace/TracerSubProcessManager.cs

```diff
@@ -136,5 +180,18 @@ private static Task StartProcessWithKeepAlive(string path, string args)
                     }
                 });
         }
+
+        private class SubProcessMetadata
+        {
+            public string Name { get; set; }
+
+            public Process Process { get; set; }
+
+            public Task KeepAliveTask { get; set; }
+
+            public string ProcessPathKey { get; set; }
```

Comment:
Can we change these to `ProcessPath` and `ProcessArguments` instead of the keys? That way the process manager doesn't need to concern itself with where the settings come from.